web/sharings/sharings.go

```diff
@@ -733,7 +733,10 @@ func checkRevokePermissions(c echo.Context, ins *instance.Instance, sharing *sha
 		return ins.PickKey(token.Claims.(*permissions.Claims).Audience)
 	}, &claims)
 	if err != nil {
-		return err
+		return sharings.ErrForbidden
+	}
+	if claims.Issuer != ins.Domain || claims.Expired() {
```

Comment:
@jinroh These tests might be done already:
1. we call `perm.GetPermissions` [here](https://github.com/cozy/cozy-stack/blob/master/web/sharings/sharings.go#L726)
1. which calls `extract` [here](https://github.com/cozy/cozy-stack/blob/master/web/permissions/extract.go#L138)
1. which calls `ParseJWT` [here](https://github.com/cozy/cozy-stack/blob/master/web/permissions/extract.go#L128)
1. which finally does the same check [here](https://github.com/cozy/cozy-stack/blob/master/web/permissions/extract.go#L67-L74)

I still think the PR is okay as these tests are done indirectly :)